Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:11-jre-slim
+
+COPY docker/entrypoint.sh /usr/local/bin/
+RUN chmod a+x /usr/local/bin/entrypoint.sh
+
+WORKDIR /usr/local/fcrepo-camel-toolbox
+
+ARG FCREPO_CAMEL_TOOLBOX_VERSION=6.0.0-SNAPSHOT
```

Comment:
I would probably omit the value here since 6.0.0 is happening right now, but could be 6.0.1 soonish'